docs/devices-api/more-info/react-native.mdx

```diff
@@ -4,8 +4,9 @@ description: "A library to integrate with Metriport on React Native - including
 ---
 
 <Info>
-  Before getting started, visit the [Apple Health docs](/devices-api/more-info/apple-health#initial-setup) and
-  follow the Initial Setup section to configure HealthKit."
+  Before getting started, visit the [Apple Health
+  docs](/devices-api/more-info/apple-health#1-enable-healthkit) and follow the Enable HealthKit
+  section to configure HealthKit."
```

Comment:
Likewise, text focused on AppleHealth but one could be using ReactNative and not connecting to AppleHealth?